query/reverse.js

```diff
@@ -9,7 +9,12 @@ function generate( params ){
     lon: params.lon
   };
 
-  var query  =  queries.distance( centroid, { size: params.size || 1 } );
+  var query = queries.distance( centroid, {
+    size: params.size || 1,
+    sort: true,
+    distance: '500km'
```

Comment:
hmmm... to be honest I am confused on this one myself.

If we set the `distance` to say 500km (a solid half day drive) it *could* potentially still not show your final destination, causing confusion that could be resolved only really by using `no geo bias` (slower queries and listing many far away places for poorly specified inputs)

The original idea was that if someone sent their lat/lon to the service it would mean that they would like to get results biased towards their location. I'm not sure if this is valid, at least not always true.

We should encourage clients to send the device location when it is available.

Breaking away from the computer and code for a second I think it should work like this.

1) When you specify a query such as `McDonald's` (with no comma, or other admin info)
2) Pelias should default that admin bias to the city in which you reside (so for me Berlin, for you NYC) -- how strong this bias is will need to be balanced
3) When the user types a comma and provides an admin segment (eg. `McDonalds, London`) then the device lat/lon is no longer considered as the user has specified their desired geo bias

thoughts?